apps/Dataman/src/screens/Receipt.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { View, Text, List } from "@ant-design/react-native";
+import { StyleSheet, ScrollView } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import { Navbar } from "../components/Navbar";
+import { InputField } from "../components/InputField";
+import { PageHeader } from "../components/PageHeader";
+import { FormButton } from "../components/Button";
+
+const Receipt = () => {
+  const Item = List.Item;
+  return (
+    <View style={{ height: "100%" }}>
+      <Navbar />
+      <PageHeader text="Good Receipt Entry" />
+      <ScrollView>
+        <List>
+          <Item arrow="horizontal">
```

Comment:
refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

Reply:
Done